user_guide_src/source/libraries/sessions.rst

```diff
@@ -545,6 +552,22 @@ Instead, you should do something like this, depending on your environment:
     chmod 0700 /<path to your application directory>/writable/sessions/
     chown www-data /<path to your application directory>/writable/sessions/
 
+Deleting Expired Session Files
```

Comment:
Apologies, I misunderstood - I was under the impression that the user guide already included a note about using cron to clean up session data when using the `FileHandler`. Please remove this section - sorry for that.

All the handlers have a garbage collection `gc()` method. Some, such as Redis or Memcached, don't need them because the session is automatically deleted when the `sessionExpiration` time is reached.

Reply:
This means there’s a 1 in 1000 chance on each request that PHP will run garbage collection to remove session files that haven’t been accessed in the last 1440 seconds. Quite a low chance, to be honest. The default is probably a 1% chance.

Maybe for the `FileHandler` we should actually add a note... something like:

> While `Redis` and `Memcached` handlers automatically clean up expired sessions by design, this isn't the case with the default `FileHandler`. PHP uses a probabilistic garbage collection system - meaning session cleanup only runs occasionally, based on chance. If your application doesn’t receive many requests, old session files may accumulate over time.
For more reliable cleanup when using `FileHandler`, consider setting up a cron job to remove expired session files.